content/docs/pulumi-cloud/reference/_index.md

```diff
@@ -1,13 +1,14 @@
 ---
-title: Reference
+title: Pulumi Cloud Reference
 title_tag: "Pulumi Cloud: Reference"
-meta_desc: This is a meta description that's hopefully just long enough to appease the linter.
+meta_desc: Learn how to interact with Pulumi Cloud programmatically using the REST API for automation and integration.
 menu:
     cloud:
-        name: Reference
```

Comment:
@interurban Why change this? I would like to keep this as **Reference** across the board (and I thought it's what we agreed on). "Pulumi Cloud" is redundant, as the whole section is about Pulumi Cloud. This will be the case everywhere.